modules/ensemblvep/main.nf

```diff
@@ -1,12 +1,13 @@
 process ENSEMBLVEP {
+    tag "$meta.id"
     label 'process_medium'
 
     conda (params.enable_conda ? "bioconda::ensembl-vep=104.3" : null)
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        !task.ext.container_tag ?
+    container "${ task.ext.container_tag ?
+            "nfcore/vep:${task.ext.container_tag}" :
+            workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
             'https://depot.galaxyproject.org/singularity/ensembl-vep:104.3--pl5262h4a94de4_0' :
-            'quay.io/biocontainers/ensembl-vep:104.3--pl5262h4a94de4_0' :
-            "nfcore/vep:${task.ext.container_tag}" }"
+            'quay.io/biocontainers/ensembl-vep:104.3--pl5262h4a94de4_0' }"
```

Comment:
@FriederikeHanssen it's here

Reply:
ok, this annotation is hard to read for me 🙈

Reply:
I wrote it, but it's hard for me too, this is probably my 3rd PR to fix these modules

Reply:
Wouldn't it be better to override the `container` directive in a config instead?
```
if ( params.use_vep_cache ){
    process {
        withName: 'ENSEMBLVEP' {
            container = "nfcore/vep:XXX"
        }
    }
}
```

Reply:
I'm now both a bit annoyed and happy because it looks like such a wonderful solution, I'll make another PR for that...